src/components/action-group/action-group.tsx

```diff
@@ -59,6 +59,8 @@ export class ActionGroup
 
   /**
    * Indicates the layout of the component.
+   *
+   * @deprecated Use the `layout` property on the component's parent component instead.
```

Comment:
Suggestion: ``Use the `layout` property on the component's parent instead.``

cc @geospatialem